Mathlib/LinearAlgebra/CrossProduct.lean

```diff
@@ -56,55 +56,70 @@ def crossProduct : (Fin 3 → R) →ₗ[R] (Fin 3 → R) →ₗ[R] Fin 3 → R :
   · intros
     simp_rw [smul_vec3, Pi.smul_apply, smul_sub, mul_smul_comm]
 
-@[inherit_doc] scoped[Matrix] infixl:74 " ×₃ " => crossProduct
+@[inherit_doc] scoped[Matrix] infixl:74 " ⨯ " => crossProduct
+
+namespace Matrix
+/-- A deprecated notation for `⨯`. -/
+@[deprecated «term_⨯_» (since := "2025-07-11")]
+scoped syntax:74 (name := _root_.«term_×₃_») term:74 " ×₃ " term:75 : term
```

Comment:
Can another reviewer confirm that these 74 and 75s are what `infixl` generates?

Reply:
This does correspond to `infixl:74`